### ISSUE TITLE
Task Scheduler Query and Not Starting Automatically.

### ISSUE THREAD

**Issue by PH03N1X1000** [UnamSanctam/SilentCryptoMiner#2153]
I wanted to ask that how does this miner auto starts itself ? Does it use task Scheduler or any registry tweaks do the trick.
In one computer i as testing there was this error "Windows 10 Issue with Task Scheduler 'MCC has detected an error in a snap-in and will unload it" and my guess is that it stopped the Task Scheduler from running and with that the miner was also not working like it would run for 15 minutes and then it would close and also not restart after a system reboot and to run it again it has to be installed again (i assume), What is all this it does not happen in any of the many computers i have tested everything works fine.
Please Help

**Comment by UnamSanctam** [comment]
In the new version it uses a Windows Service for autostarting as administrator, the MCC gets disabled to hide the miner from the event logs.

**Comment by PH03N1X1000** [comment]
> In the new version it uses a Windows Service for autostarting as administrator, the MCC gets disabled to hide the miner from the event logs.

Ooh that information is very useful thanks. So then can you tell me what are the probable causes of miner not running after the reboot.
And does this program tweak the task scheduler in any way ?

**Comment by UnamSanctam** [comment]
>Ooh that information is very useful thanks. So then can you tell me what are the probable causes of miner not running after the reboot.

There might be a problem if you have a space in the name of the file when trying to start it for the first time, it's a bug with the powershell string only on some computers. You can run the checker after installation to see if the miner and watchdog are running. It's also possible that that computer has something like deepfreeze, which resets the system to a previous state on restart. It depends on what kind of computer it's not working on.

>And does this program tweak the task scheduler in any way ?

The latest version never touches the Task Scheduler in any way.

**Comment by PH03N1X1000** [comment]
The file is named as Trial.exe and i have used the checker file in the victim PC which does not show the miner as running which is surprising because i have installed it 2 times in 2 days on the same computer.

**Comment by PH03N1X1000** [comment]
Yep it shows up just fine and everything is normal but the problem is after the reboot of system it stops.

**Comment by UnamSanctam** [comment]
Is the service still there if you check the services after restart? The computer could have something like deepfreeze like I said before, meaning that the system might reset to a previous state on every restart.

**Comment by PH03N1X1000** [comment]
The Trial.exe files gets deleted after it executes itself can you give any other clue through which i can check the service and its very unlikely to have a deepfreeze software installed but the probablity is not 0 i will do some more recon and post here soon
Thanks
